docs/prerequisites.md

```diff
@@ -1,5 +1,13 @@
 # Prerequisites
 
+## Node name conventions
+
+AWS supports two naming conventions: [IP-based or resource-based naming](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-naming.html).
```

Comment:
Maybe an additional sentence for further clarity, something like: `In order to use either of the naming conventions, simply name the nodes in your cluster following either of the two naming conventions.  Both naming conventions can be used simultaneously in the same cluster.` (they can right?)